sunkit_image/time_lag.py

```diff
@@ -138,8 +137,17 @@ def _dask_check(signal, lags):
         return lags
     if isinstance(signal, dask.array.Array):
         return dask.array.from_array(lags, chunks=lags.shape)
-    else:
-        return lags
+    return lags
+
+
+def _deal_with_dask(array):
```

Comment:
A hack.

Reply:
I am very very 👎  on this. This defeats the purpose of using putting a Dask array in here at all.